Fix UPE appearance on block themes: background color and Bunny Fonts (#5187)
woocommerce/woocommerce-gateway-stripe: changelog.txt

*** Changelog ***

= 10.6.0 - xxxx-xx-xx =

> * Fix - Resolve incorrect Stripe PE background color on block themes by adding missing background selectors and replacing defunct fast.fonts.com with fonts.bunny.net

Reviewer: Nit: I think we can simplify this changelog entry to be more merchant-friendly.
```suggestion
* Fix - Better background color detection for block themes and allow fonts from fonts.bunny.net
```
